src/main/kotlin/app/CacheService.kt

```diff
@@ -0,0 +1,85 @@
+@file:Suppress("SqlResolve")
+
+package app
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import org.slf4j.LoggerFactory
+import java.sql.DriverManager
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+
+object CacheService {
+    private const val urlToDb = "jdbc:h2:mem:userinfo"
+    private val log = LoggerFactory.getLogger(CacheService.javaClass)
+    private val objectMapper = jacksonObjectMapper()
+
+    private fun createTableIfAbsent() {
+        val connection = DriverManager.getConnection(urlToDb)
+        val statement = connection.createStatement()
+
+        statement.execute(
+            "CREATE TABLE IF NOT EXISTS userinfo (" +
+            "id VARCHAR2 PRIMARY KEY," +
+            "timestamp TIMESTAMP WITH TIME ZONE, " +
```

Comment:
is there anything special that needs to be done for H2 to store in UTC?

Reply:
No. I blindly chose it at first and forgot about that. 😁
[849c6e4](https://github.com/tipsy/profile-summary-for-github/pull/113/commits/849c6e4557567260c768ef8c5ff6b6c08f3dcce1)